packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -93,10 +95,14 @@ class SnackBarAction extends StatefulWidget {
     required this.label,
     required this.onPressed,
   }) : assert(label != null),
-       assert(onPressed != null);
+      assert(onPressed != null);
 
   /// The button label color. If not provided, defaults to
   /// [SnackBarThemeData.actionTextColor].
+  ///
+  /// If [textColor] is a [MaterialStateColor], then the effective text
+  /// color can depend on the [MaterialState.selected] state, e.g. if the
+  /// action is hovered or not.
```

Comment:
Not sure how to word this better, but perhaps:

```suggestion
  /// If [textColor] is a [MaterialStateColor], then the text color will be 
  /// be resolved against the set of [MaterialState]s that the action
  /// text is in, thus allowing for different colors for pressed, hovered, etc.
  ```

Reply:
Done - tweaked the ending a little